docs/contributor/05-10-e2e_tests.md

```diff
@@ -126,6 +126,31 @@ The test executes the following steps:
     make skr-networking-test
     ```
 
+## Binding Tests
```

Comment:
I wonder whether it should be `service binding` here and in line 135? If just `binding` is sufficient, please ignore this comment.

Reply:
I will leave it as it is. It's documentation for the tests, so I think that's sufficient.